AsyncDisplayKit/Details/ASChangeSetDataController.mm

```diff
@@ -81,8 +81,16 @@ - (void)endUpdatesAnimated:(BOOL)animated completion:(void (^)(BOOL))completion
     
     ASDataControllerLogEvent(self, @"triggeredUpdate: %@", _changeSet);
     
+    // TODO: move
+    [super prepareMoveItemChanges:[_changeSet itemChangesOfType:_ASHierarchyChangeTypeMove]];
+    
+    // Prepare UIKit batch updates
     [super beginUpdates];
     
+    for (_ASHierarchyMoveItemChange *change in [_changeSet itemChangesOfType:_ASHierarchyChangeTypeMove]) {
+      [super moveRowAtIndexPath:change.fromIndexPath toIndexPath:change.toIndexPath withAnimationOptions:change.animationOptions];
+    }
+    
```

Comment:
It sounds more like a rename to the method, and I understand there might be ambiguity. I will change it, but some clarifications go on.

Yet, since I'm using the existing logic to insert the `move-to` nodes with the new node block at the toIndexPaths of the updated data source, there is no need to store them in a dictionary and reuse.

In fact, I think as move is decomposed to `delete` and `insert`, they shall be processed together with other deletes and inserts in desc and asc ways respectively. If we isolate move-related methods like the one you suggested, it might break the rules of batching.

What do you think?